.coderabbit.yaml

```
@@ -0,0 +1,90 @@
+# CodeRabbit Configuration
+# Auto-resolves outdated comments and waits for CI/CD checks before reviewing
+
+# Review configuration
+reviews:
+  # Automatically resolve outdated comments when code changes
+  auto_resolve_outdated_comments: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)